src/Search/Console.php

```diff
@@ -51,6 +54,17 @@
         ],
     ];
 
+    public function queueCreateCommand()
+    {
+        /** @var SqsClient $queue */
```

Comment:
annotate/tag/describe this as development-only: queues should never be created manually on the real AWS